girder/cluster_filesystem/plugin_tests/constants.py

```diff
@@ -0,0 +1,94 @@
+import stat
+
+CLUSTER_LOAD = {
+    '_id': 'dummy',
+    'type': 'newt',
+    'name': 'eyeofnewt',
+    'config': {
+        'host': 'cori'
+    }
+}
+
+PATH = '/a/b/c'
+
+DIR_ID_FIELDS = {
+    'clusterId': 'dummy',
+    'path': PATH
+}
+
+CURR_DIR = {
+    'name': '.',
+    'group': 'group',
+    'user': 'user',
+    'mode': stat.S_IFDIR,
+    'date': 'Feb  7 16:26',
+    'size': 0
+}
+
+PARENT_DIR = {
+    'name': '..',
+    'group': 'group',
+    'user': 'user',
+    'mode': stat.S_IFDIR,
+    'date': 'Feb  7 16:26',
+    'size': 0
+}
+
+DIR1 = {
+    'name': 'dir1',
+    'group': 'group',
+    'user': 'user',
+    'mode': stat.S_IFDIR,
+    'date': 'Feb  7 16:26',
+    'size': 0
+}
+
+DIR2 = {
+    'name': 'dir2',
+    'group': 'group',
+    'user': 'user',
+    'mode': stat.S_IFDIR,
+    'date': 'Feb  7 16:26',
+    'size': 0
+}
+
+DIR3 = {
+    'name': 'dir3',
+    'group': 'group',
+    'user': 'user',
+    'mode': stat.S_IFDIR,
+    'date': 'Feb  7 16:26',
+    'size': 0
+}
+
+FILE1 = {
+    'name': 'file1',
+    'group': 'group',
+    'user': 'user',
+    'mode': stat.S_IFREG,
+    'date': 'Feb  7 16:26',
+    'size': 123
+}
+
+FILE2 = {
+    'name': 'file2',
+    'group': 'group',
+    'user': 'user',
+    'mode': stat.S_IFREG,
+    'date': 'Feb  7 16:26',
+    'size': 456
+}
+
+FILE3 = {
+    'name': 'file3',
+    'group': 'group',
+    'user': 'user',
+    'mode': stat.S_IFREG,
+    'date': 'Feb  7 16:26',
+    'size': 78910
+}
+
+FILE_ID_FIELDS = {
+    'clusterId': 'dummy',
+    'path': "{}/{}".format(PATH, FILE1['name'])
```

Comment:
Can we use %s string formatting here.

Reply:
We could, but why? %s is the old formatting style, and is a little less flexible than .format().
See https://pyformat.info/

Reply:
We are using % style in the rest of the code base, so for better or worse I would favor consistency.